### PR TITLE
Check Docstrings Exist

### LINKED ISSUE #284: [Testing]: check docstrings for consistency
### What would you like to see added to NeuroConv?

Replaces the dynamic parsing proposal of #278

Add simple tests that check the docstring description of both upstream arguments that are copied and propagated as well as internal repeated arguments

very low priority, however

### Is your feature request related to a problem?

_No response_

### Do you have any interest in helping implement the feature?

Yes.

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://github.com/catalystneuro/neuroconv/blob/main/.github/CODE_OF_CONDUCT.rst)
- [X] Have you ensured this bug was not already [reported](https://github.com/catalystneuro/neuroconv/issues)?

### PR DESCRIPTION
**Pull request by pauladkisson**
Fixes #284 pt 2

Awaiting https://github.com/catalystneuro/.github/pull/19

### CODE REVIEW

File: .github/workflows/check_docstrings.yaml [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+
+jobs:
+  check-docstrings:
+    runs-on: ubuntu-latest
+    steps:
+      - id: check_docstrings
+        uses: catalystneuro/.github/.github/actions/check_docstrings@main
+        with:
+          python-version: '3.10'
+          repository: 'catalystneuro/neuroconv'
```
Comment (h-mayorquin) [inline .github/workflows/check_docstrings.yaml:14]: Can you use the `${{github.repository}}` context?

**Review by h-mayorquin — COMMENTED** [review]
Oh, taking a look at the referenced PR.

**Comment by h-mayorquin** [comment]
Question:
@CodyCBakerPhD @pauladkisson 

Would having all of the following linter rules in ruff enabled be equivalent to this:
image

https://docs.astral.sh/ruff/rules/#pydocstyle-d

**Comment by pauladkisson** [comment]
Yes, I think using ruff with skips makes this PR obsolete.

**Pull request closed without merging**